### ISSUE TITLE
MoveEditor{Left,Right,Up,Down}InGroup shortcuts don't work for "Settings" and "Keyboard Shortcuts" tabs 

### ISSUE THREAD

**Issue by ldorigo** [microsoft/vscode#129958]
Issue Type: <b>Bug</b>

I'm not sure if this is a bug or just bad UX, but currently, if I assign a keyboard shortcut to move the current editor within the current group (e.g. workbench.action.moveEditorToLeftGroup), it only works on text-editing tabs - not on the settings tab or the keyboard shortcuts tab. 

Not a major issue,  but it breaks keyboard based workflows since the only way to move the settings or Keyboard Shortcuts tabs is with the mouse.

VS Code version: Code 1.58.2 (c3f126316369cd610563c75b1b1725e0679adfb3, 2021-07-14T22:30:16.440Z)
OS version: Linux x64 5.10.54-1-lts
Restricted Mode: No
Remote OS version: Linux x64 5.8.0-63-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 3700X 8-Core Processor (16 x 4230)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|31.36GB (0.50GB free)|
|Process Argv|--crash-reporter-id 66ea3ec9-05eb-4edd-8d35-34bbe6c1aba1|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|plasma-i3|
|XDG_CURRENT_DESKTOP|KDE|
|XDG_SESSION_DESKTOP|KDE-i3|
|XDG_SESSION_TYPE|x11|

|Item|Value|
|---|---|
|Remote|SSH: medrel|
|OS|Linux x64 5.8.0-63-generic|
|CPUs|Intel(R) Xeon(R) CPU E3-1220 v5 @ 3.00GHz (4 x 839)|
|Memory (System)|15.56GB (11.29GB free)|
|VM|0%|
</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
docs-view|bie|0.0.9
systemd-unit-file|coo|1.0.6
monkey-patch|ioc|0.1.15
remote-containers|ms-|0.187.1
remote-ssh|ms-|0.65.7
remote-ssh-edit|ms-|0.65.7
cython|tcw|0.1.0
vscode-icons|vsc|11.6.0
vim|vsc|1.21.5
xml|Dot|2.5.1
python|ms-|2021.7.1060902895
vscode-pylance|ms-|2021.7.7
jupyter|ms-|2021.8.1054968649

(9 theme extensions excluded)

</details>

**Comment by bpasero** [comment]
@lramos15 maybe related to our work in editor land, haven't investigated yet.

**Comment by ldorigo** [comment]
Apparently the shortcuts also don't work for native notebook tabs.

**Comment by bpasero** [comment]
I cannot reproduce, for me the editor moves to the left group (both settings and keyboard shortcuts).

Demo:

![recording](https://user-images.githubusercontent.com/900690/128153010-8efc2984-bb99-43f9-8649-8a7ce5bdc122.gif)

**Comment by ldorigo** [comment]
On my phone now so can't lookup the exact names or post screenshots,  but its the shortcuts for moving tabs *within* groups, not across groups

**Comment by bpasero** [comment]
Also works for me, I can move settings editor to the right.

**Comment by ldorigo** [comment]
Just tried again on a different computer (and OS) and still does not work. 

Are you sure we are talking about the same command? See screenshots below.

![image](https://user-images.githubusercontent.com/8631181/128337684-c6653209-80d7-49a9-96b3-c31b29b157bf.png)

![image](https://user-images.githubusercontent.com/8631181/128337939-afd88bcc-0930-463d-ad49-72759a3c2fad.png)

Note that it also doesn't work when invoking the command from the command palette, so it's not an issue with conflicting keyboard shortcuts,

**Comment by ldorigo** [comment]
Screencast for proof :-) 

![Peek 2021-08-05 13-16](https://user-images.githubusercontent.com/8631181/128341581-55c0af9c-ea83-4680-a746-345a1fd781ce.gif)

**Comment by lramos15** [comment]
I'm still not seeing this. Here's a screencap to ensure I'm doing the same steps you are. Please also try disabling all extensions 
![recording (3)](https://user-images.githubusercontent.com/4544166/128362518-7b0b0f18-466f-42a7-998d-e233c9c378ef.gif)

**Comment by bpasero** [comment]
Is it possible that this only reproduces when trying via keybinding?

**Comment by lramos15** [comment]
> Is it possible that this only reproduces when trying via keybinding?

I did keybindings as well and it works. @ldorigo also says it happens regardless

**Comment by bpasero** [comment]
Bump

**Comment by ldorigo** [comment]
Hi, sorry for the latency.
I'm unsure whether this somehow got fixed in 1.59 or I was doing something wrong, but now I am able to move editors within groups using the command in the command palette.

Regarding doing it via keybindings, I found that I had a "when" condition stating "editorFocus". Removing the condition makes it work. I would like to be able to only have those shortcuts trigger when I am focusing a tab (and not other parts of the UI) - I thought editorFocus was the right condition, but apparently "editors" are exclusively text editors (not settings/keyboard shortcuts or notebooks)? If that's expected behaviour, then feel free to close this (although you might benefit from clearer naming/docs :-))
